scripts/upload-genesis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+GENESIS_DIR="/tmp/coda_cache_dir/"
+
+# Need AWS creds to upload
+if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+    echo "WARNING: AWS_ACCESS_KEY_ID is missing, not uploading files for genesis"
+else
+    ls -l ${GENESIS_DIR}genesis_* && \
```

Comment:
This needs to be changed to `coda_genesis_*` too

Reply:
Done.

Reply:
🤦 thank you